### PR TITLE
Release 7.5.4

### PR DESCRIPTION
**Pull request by rlepinski**
Adds Java6 CI check and an option to disable the GoogleServicesPlugin if another plugin is also adding it.

Doing a release since the README is being updated with the new gradle option.

### CODE REVIEW

**Comment by marc-scig** [comment]
🖼

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,10 @@
+==============================
+Version 7.5.4 - April 18, 2019
+==============================
+Added a gradle property `uaSkipApplyGoogleServicesPlugin` that will disable applying
+the `GoogleServicesPlugin` if set to `true`. This option should only be used if another
+plugin also applies the `GoogleServicesPlugin` to avoid build errors.
+
```
Comment (BrianBatchelder) [inline CHANGELOG]: Maybe add something regarding the Java6 CI check?

Comment (rlepinski) [reply CHANGELOG]: Thats irrelevant information for our users.

**Comment by marc-scig** [comment]
👍

**Pull request merged**